docs/sdk-and-tools/indices/blocks.md

```diff
@@ -0,0 +1,93 @@
+---
+id: blocks
+title: blocks
+---
+
+The `_id` field for this index is composed of block hash hex encoded
+
+| Field                 | Description                                                                                                |
+|-----------------------|------------------------------------------------------------------------------------------------------------|
+| nonce                 | The nonce field represents sequence number of block.                                                       |
+| round                 | The round field represents the round when block was proposed and executed.                                 |
+| epoch                 | The epoch field represent the epoch when the block was proposed and executed.                              |
+| miniBlocksHashes      | The miniBlocksHashes field represents the hashed hex encoded of the miniblocks that was included in block. |
+| miniBlocksDetails     | The miniBlocksDetails field represents the details of all the miniblocks that was included in block.       |
+| notarizedBlocksHashes | The notarizedBlocksHashes field represents the hashes of the blocks the was notarized in current block.    |
+| proposer              | The proposer field represents the index of the validator that proposed this block.                         |
+| validators            | The validators field represents the indices on the validators that have signed for this block.             |
+| pubKeyBitmap          | The pubKeyBitmap field represents the pub key bitmap.                                                      |
+| size                  | The size field represents the size of block in bytes.                                                      |
+| sizeTxs               | The sizeTxs field represents the size of transactions from block in bytes.                                 |
+| timestamp             | The timestamp field represents the timestamp when block was proposed and executed.                         |
+| stateRootHash         | The stateRootHash field represents the state root hash of tree when block was proposed and executed.       |
+| prevHash              | The prevHash field represents the hash of the previous block.                                              |
+| shardId               | The shardId field represents the shard id of block.                                                        |
```

Comment:
The shardId field represents the shard where the block was executed.

Reply:
done